### PR TITLE
VACMS-6629: Moving vamc logic from va gov backend module file to vamc…

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
**Relates to #6629** 
 - moves code that is exclusive to VAMC product to `va_gov_vamc` from `va_gov_backend`

## Testing done
Visual / behat

## QA steps
 - [ ] General site inspection: visit multiple vamc content add forms, and ensure errors / notices are not present:
   - [ ] `node/add/health_care_local_facility`
   - [ ] `node/add/health_care_local_health_service`
   - [ ] `node/add/full_width_banner_alert`
   - [ ] `node/add/health_care_region_page`

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -357,7 +357,7 @@
                 "3205691 - Disallow aria-required on fieldsets": "https://www.drupal.org/files/issues/2021-03-25/disallow-aria-required-fieldsets.patch",
                 "3229493 - TextareaWidget::errorElement() does not consult violation property path.": "https://www.drupal.org/files/issues/2021-08-23/test_module-3229493-2.patch",
                 "3027240 - Undefined index: #parents FormState.php": "https://www.drupal.org/files/issues/2019-01-21/form_system-undefined_index-3027240-2.patch",
-                "3213473 - Checkbox input can't have aria-required attribute": "https://www.drupal.org/files/issues/2021-11-05/3213473-36.patch"
+                "3213473 - Checkbox input can't have aria-required attribute": "https://www.drupal.org/files/issues/2021-11-07/3213473-39.patch"
```
Comment (swirtSJW) [inline composer.json:360]: This all can go away.

**Review by swirtSJW — APPROVED** [review]
This will be ready to go after conflict resolution.

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
I fixed the conflict.  Just waiting on tests to complete.

**Comment by ethanteague** [comment]
> I fixed the conflict. Just waiting on tests to complete.

ty buddy

**Pull request merged**